swagger_server/controllers/respondent_controller.py

```diff
@@ -0,0 +1,34 @@
+import connexion
```

Comment:
controllers is auto-generated and overridden. Without recompiling the java app and changing the templates we can't do much about this.